python/py-parameterized/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-parameterized
+version             0.7.0
+revision            0
+
+categories-append   testing
```

Comment:
Does this make sense or not?

Reply:
I am under the impression that you should only add categories that are already present in MacPorts...

Reply:
That would generally be my impression as well.

The thing is ... we sometimes need to introduce new categories as this is a fluid thing. But it definitely doesn't make any sense to have just one port in a category. We would need to add a bunch of other unit / ci testing utilities there, else it doesn't make much sense.